docs/metrics/bleu.md

```diff
@@ -0,0 +1,132 @@
+
+    | Score | Qualitative Interpretation |
+    | ---   | ---                        |
+    | < 0.1 | Useless output             |
+    | 0.1 - 0.4 | Varies in quality; May not be acceptable |
+    | 0.4 - 0.6 | High quality generated text |
+    | > 0.6 | Better than human quality  |
+
+2. **Track Trends Over Time**: Rising scores signal improvements in models, while drops could hint at issues or changes in your dataset.
+
+3. **Combine It With Other Metrics**: BLEU primarily measures n-gram overlap, overlooking some nuancies like context and understanding. While a high BLEU score is promising, it doesn't guarantee flawless text. A complementary metric like [BertScore](bertscore.md) may help in quantifying your model's performance from other perspectives.
+
+## Example
+
+**Generated Sentence**: Kolena is an ML testing and debugging platform to find hidden model behaviors and demystify model development <br>
+**Reference Sentence**: Kolena is a comprehensive machine learning testing and debugging platform to surface hidden model behaviors and take the mystery out of model development
+
+??? example "Step 1: Tokenization & n-Grams"
+    Splitting our sentences up into 1-, ..., 4-grams, we get:
+
+    **Generated Sentence**:
+
+    | n | n-Grams |
+    | ---   | ---                        |
+    | 1 | ["kolena", "is", "an", "ml", "testing", "and", "debugging", "platform", "to", "find", "hidden", "model", "behaviors", "and", "demystify", "model", "development"]|
+    | 2 | ["kolena is", "is an", "an ml", "ml testing", "testing and", "and debugging", "debugging platform", "platform to", "to find", "find hidden", "hidden model", "model behaviors", "behaviors and", "and demystify", "demystify model", "model development"]|
+    | 3 | ["kolena is an", "is an ml", "an ml testing", "ml testing and", "testing and debugging", "and debugging platform", "debugging platform to", "platform to find", "to find hidden", "find hidden model", "hidden model behaviors", "model behaviors and", "behaviors and demystify", "and demystify model", "demystify model development"] |
+    | 4 | ["kolena is an ml", "is an ml testing", "an ml testing and", "ml testing and debugging", "testing and debugging platform", "and debugging platform to", "debugging platform to find", "platform to find hidden", "to find hidden model", "find hidden model behaviors", "hidden model behaviors and", "model behaviors and demystify", "behaviors and demystify model", "and demystify model development"]  |
+
+    **Reference Sentence**:
+
+    | n | n-Grams |
+    | ---   | ---                        |
+    | 1 | ["kolena", "is", "a", "comprehensive", "machine", "learning", "testing", "and", "debugging", "platform", "to", "surface", "hidden", "model", "behaviors", "and", "take", "the", "mystery", "out"]|
+    |...|
```

Comment:
I would prefer to add all n-Grams even for the reference sentence if we go with a shorter sentence.